templates/bundler_audit.rake

```diff
@@ -4,9 +4,25 @@ if Rails.env.development? || Rails.env.test?
   namespace :bundler do
     desc "Updates the ruby-advisory-db and runs audit"
     task :audit do
+      prepare_bundler_audit_to_exit_with_error_if_update_fails
       %w(update check).each do |command|
         Bundler::Audit::CLI.start [command]
       end
     end
   end
+
+  def prepare_bundler_audit_to_exit_with_error_if_update_fails
```

Comment:
`update_bundler_audit_or_exit_with_error?` is a clearer name if the method is changed from patching `update` to running `Database.update!` etc. With patching, the code can take advantage of `say "...", :pretty_colors`.